src/collection/account.service.ts

```diff
@@ -354,25 +354,40 @@ export class AccountService {
           );
         const projectName = projectDtoArr[0].collection.name;
         try {
-          this.auditService.recordToolsSync(
-            'start',
-            'unknown',
-            `Start sync: ${service.collection.scmUrl}`,
-            projectName,
-            serviceName,
-          );
-          await this.githubService.refresh(
-            projectName,
-            serviceName,
-            service.collection.scmUrl,
-          );
-          this.auditService.recordToolsSync(
-            'end',
-            'success',
-            `End sync: ${service.collection.scmUrl}`,
-            projectName,
-            serviceName,
-          );
+          const githubRegex = /^https:\/\/github\.com[:/](.+?)\/(.+?)(\.git)?$/;
```

Comment:
This should be the default, but, we should be able to define a different one as. Our regex probably should be like: ^https://github.com/(bcgov(-nr|-c)?/[a-zA-Z0-9_-]+)$